docs/bump.md

```diff
@@ -76,18 +77,30 @@ optional arguments:
                         choose type of prerelease
   --increment {MAJOR,MINOR,PATCH}
                         manually specify the desired increment
+  --check-consistency, -cc
+                        check consistency among versions defined in commitizen
+                        configuration and version_files
 ```
 
-## Configuration
-
-### `changelog`
+### `--changelog`
 
 Generate a **changelog** along with the new version and tag when bumping.
 
 ```bash
 cz bump --changelog
 ```
 
+### `--check-consistency`
```

Comment:
I would add an example situation

Reply:
Yep, that would much eaiser to understand. I just update it 🎉